docs/main/user-guides/observability.md

```diff
@@ -0,0 +1,273 @@
+---
+id: observability
+title: Observability
+keywords:
+ - Monitoring
+ - Observability
+ - Pool metrics
+ - Prometheus
+ - Grafana
+description: This guide explains about monitoring your cluster.
+---
+# Observability
+
+Monitoring your cluster is essential to ensure the health, efficiency, and reliability of your storage infrastructure.​ The following technologies are implemented to monitor your cluster:
+
+- Prometheus to collect data
+- Grafana to visualize your data
+
+Prometheus and Grafana, together provide a robust solution for collecting metrics, visualizing data, and creating alerts.
+
+## Monitoring Setup
+
+OpenEBS provides a basic cloud-native monitoring stack built using Prometheus and Grafana, as an add-on Helm chart. This has pre-configured dashboards for visualization of metrics from the various OpenEBS storages.
+
+:::info
+A set of Grafana dashboards for monitoring and visualization of exported metrics via various OpenEBS storage are bundled with OpenEBS installation which is by default **disabled**. Grafana will be using Prometheus as a data source.
```

Comment:
this part is not true currently, as it is removed from umbrella chart of openebs installation.